python/sglang/srt/mem_cache/storage/mooncake_store/mooncake_store.py

```diff
@@ -171,6 +177,52 @@ def __init__(self, storage_config: HiCacheStorageConfig = None):
             logger.error("An error occurred while loading the configuration: %s", exc)
             raise
 
+    def check_server(self):
+        master_server_ip = self.config.master_server_address.split(':')[0]
+        health_url = f"http://{master_server_ip}:9003/health"
+        segments_url = f"http://{master_server_ip}:9003/get_all_segments"
+        start_time = time.time()
+
+        health_result = False
+        segments_result = False
+        while time.time() - start_time < SETUP_TIMEOUT:
+            try:
+                check_health_resp = requests.get(health_url)
+            except Exception as e:
+                logger.info("waiting Mooncake store master started, cost_time: %s", time.time() - start_time)
+                time.sleep(3)
+                continue
+
+            if check_health_resp.text != "OK":
+                logger.info("waiting Mooncake store master started, cost_time: %s", time.time() - start_time)
+                time.sleep(3)
+                continue
+
+            health_result = True
+
+            try:
+                check_segments_resp = requests.get(segments_url)
```

Comment:
Why need to check all segments?

Reply:
> Why need to check all segments?

Warmup will put a key-value into store, so at least a segment should be mounted before warmup.